### ISSUE TITLE
files not accessible within network; cloudflare & @font-face

### ISSUE THREAD

**Issue by cmccullough2** [cmv/cmv-app#570]
I am having a connectivity issue.  My network will not allow me to connect to:
1.  //cdnjs.cloudflare.com/ajax/libs/proj4js/2.3.12/proj4.js 
2.  CSS3114: @font-face failed OpenType embedding permission check. Permission must be Installable

Where can I reference these files within the application?  The WAB widgets will not work now.   Also, due to the same issue, I cannot post this on Gitter.

**Comment by tmcgee** [comment]
@cmccullough2 you can host everything on your own servers if desired/required.

1. The Proj4JS library is used by the MapInfo and StreetView core cmv widgets as well as some contributed widgets. At present, you would have to modify the source files for those widgets to reference a local copy of that library. Pull Request #547 which is pending a final review step proposes moving the reference to a single place for all widgets that use Proj4JS.

2. Regarding the @font-face, is that possibly from the Font-Awesome font library. You can host that locally as well.

**Comment by cmccullough2** [comment]
The issue was resolved by adding mime types .woff and .json to the server.

**Comment by cmccullough2** [comment]
I tried re-pointing the proj4js library using a suggestion from a previous thread that included the app.js file.  It did not work.  I tired a plethora of things.  Finally, the issue was resolved by adding mime types .woff and .json to the server.  I am not sure why these mime types were not on the server, but it is what it is.  I am not the web admin.  I am very happy that the widgets are working fabulously at this time.
